docs/src/openapi-v1.yaml

```diff
@@ -935,6 +935,113 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/PalletsAssetsInfo'
+    /pallets/{palletId}/consts:
```

Comment:
All the section of `/pallets/{palletId}/consts:` has an extra indentation so the [swagger editor](https://editor.swagger.io/) complaints and in the built version of the docs this endpoint is not appearing. So the indent of this section has to be decreased by one tab.